docs/UserGuide.md

```diff
@@ -92,6 +92,22 @@ Examples:
 * `findo keychain` returns `banana keychain` and `keychain`
 * `findo apple keychain` returns `apple painting`, `banana keychain`<br>
 
+### Sorting orders by time created: `sorto`
+
+Sort the displayed list of orders by the time at which they were created.
```

Comment:
Small nitpick, but to keep things consistent, make it 'Sorts' instead of just 'Sort', just like other commands.